person.rb

```diff
@@ -1,15 +1,16 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- [ ] Kindly make sure `Getters` and `Setters` for `@rentals`  are also present i.e. you should add the missing attribute value (which is `:rental`) on the `attr_accessor` and remove it from the `attr_reader`